### PR TITLE
Update data-importer how-to document.

### PR DESCRIPTION
**Pull request by mahendra-mahajan-mendix**
Initial draft of the how-to document for Data Importer. Based on the existing documentation with more details about the individual steps.

### CODE REVIEW

File: content/en/docs/howto/integration/data-importer [hunk 1]
```diff
@@ -2,3 +2,68 @@ title: "Data Importer Extension"
 url: /howto/integration/data-importer
 weight: 21
 description: "Overview of the Data Importer in Studio Pro"
```
Comment (MarkvanMents) [inline content/en/docs/howto/integration/data-importer]: Missing front matter delimiters
```suggestion
description: "Overview of the Data Importer in Studio Pro"
---
```

Also needed as the first line of the doc.

File: content/en/docs/howto/integration/data-importer [hunk 2]
```diff
@@ -71,11 +71,7 @@ The Data Importer document creation is complete and can be used to import data i
...
 ## 4 Building your App {#build-data-importer-app}
 
-Say you want to periodically import data from an Excel file which is generated by some other App or enterprise system. You would like to validate if the data is correct and can be imported with the given mapping without any errors. Let's try to build an App which will achieve this using Data Importer.
-
-But before that a little background about the Data Importer extension : It requires [run-time module](https://marketplace.mendix.com/link/component/219833) from Marketplace to help you achieve your goals. So search the Data Importer module in the *Marketplace* and import it. The DataImporter module should be visible under the Marketplace modules in your App as shown below.
-
-{{< figure src="/attachments/refguide/integration/data-importer-extension/data-importer-in-marketplace-module.png" >}}
+Use the Data Importer to periodically import and validate data from an Excel file that is generated by another app or system. 
 
 ### 4.1 Custom Activity {#Import-data-from-file}
```
Comment (quinntracy) [inline content/en/docs/howto/integration/data-importer]: Can you provide some extra context for the example process below? Is there any additional information that the user may benefit from knowing in regard to the sample?

File: content/en/docs/howto/integration/use-the-data-importer.md [hunk 3]
```diff
@@ -91,15 +91,15 @@ The following are the properties in the **Output** section:
 
 ### 4.2 Build the Pages
 
-As the *Import data from file* custom activity needs an input file from where data needs to be imported, let's build a page where a *System.FileDocument* can be uploaded and then be fed to our custom activity.
+The *Import data from file* custom activity needs an input file to import data from. The example below builds a page where a `System.FileDocument`is uploaded and fed to the custom activity.
 
-1. Add a Button on the home page with a caption 'Upload Customer Data'
-2. On click of this button you can create an Object of System.FileDocument and pass the control to another page (UploadCustomerData) where file can be uploaded.
+1. Open the home page and add a button with the caption *Upload Customer Data*.
+2. Click the button and create a `System.FileDocument` object and pass the control to another page (**UploadCustomerData**) where the file can be uploaded.
```
Comment (quinntracy) [inline content/en/docs/howto/integration/use-the-data-importer.md]: To confirm, for this example, the user would need to add a page to their app called **UploadCustomerData**, correct?

File: content/en/docs/howto/integration/use-the-data-importer.md [hunk 4]
```diff
@@ -91,15 +91,15 @@ The following are the properties in the **Output** section:
...
 
 {{< figure src="/attachments/refguide/integration/data-importer-extension/home-page-button.png" >}}
 
-3. On the 'UploadCustomerData' page - include a Data view for FileDocument and include a 'File Manager' to assist a file upload.
-4. Add an Action button to call Microflow which will help us import the input file. The uploaded file will be passed as a parameter to this Microflow.
+3. On the **UploadCustomerData** page, include a data view for the *FileDocument* and include a 'File Manager' to assist with a file upload.
+4. Add an **Action** button to call a microflow, which will import the input file. The uploaded file will be passed as a parameter to this microflow.
```
Comment (quinntracy) [inline content/en/docs/howto/integration/use-the-data-importer.md]: Should we assume the user knows how to add a parameter or should we add extra information about them?

File: content/en/docs/howto/integration/use-the-data-importer.md [hunk 5]
```diff
@@ -105,25 +105,25 @@ The *Import data from file* custom activity needs an input file to import data f
...
    {{< figure src="/attachments/refguide/modeling/integration/data-importer-extension/integration-activity.png" >}}
 
-2. As soon as the **Import data from file** activity is added into Microflow, you will start seeing 3 errors in the console; mainly complaining aobut missing input file, data importer document and Output variable being empty.
+2. When the **Import data from file** activity is added into microflow, you see three errors in the console.
 
    {{< figure src="/attachments/refguide/modeling/integration/data-importer-extension/custom-activity.png" >}}
 
-3. Let's try to address these errors. Double-click the activity and in the **File** field, choose the input file that is passed on from the file upload page to this Microflow as a parameter.
-4. In the **Data importer document** field, click *Select* and choose the Data Importer document you want to use.
+3. To address these errors, double-click the activity and in the **File** field, choose the input file that is passed from the file upload page to this microflow as a parameter.
```
Comment (quinntracy) [inline content/en/docs/howto/integration/use-the-data-importer.md]: When testing this process, I didn't have the option to choose/select a data importer document from the drop-down. Are there ways to troubleshoot this step?

**Pull request merged**